doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -210,6 +232,28 @@ The default values of the VOF parameters are given in the text box below.
           -number of wet cells: 24
           -number of peeled cells: 1
 
+.. _improve wetting:
+
+Improving the Wetting Mechanism
++++++++++++++++++++++++++++++++++++
+
+In the framework of incompressible fluids, a layer of the lowest density fluid (e.g. air) can form between the highest density fluid (e.g. water) and the boundary, preventing its wetting. Two strategies can be used to improve the wetting mechanism:
+
+1. Increase the ``diffusivity`` to the transport equation (e.g. ``set diffusivity = 1e-2``), so that the higher density fluid spreads even more to the boundary location.
+
+.. tip::
+  It is strongly advised to sharpen the interface more often (e.g. ``set frequency = 2`` or even ``1``) to limit interface blurriness due the added diffusivity. As peeling-wetting is handled after the transport equation is solved, but before interface sharpening, sharpening will not prevent the wetting from occurring.
+
+2. Remove the conservation condition on the lowest density fluid (e.g. ``set conservative fluid = fluid 1``). The mass conservation equation in the cells of interest is replaced by a zero-pressure condition, to allow the fluid to get out of the domain.
```

Comment:
unrelated to this PR, but now that we have the compressible equation of state, we should remvoe this conservative fluid shenanigan. TOOOO BEEEE DEPRECATTEEEEDDD